new-dti-website/components/apply/ApplyFAQ.tsx

```diff
@@ -40,7 +40,7 @@ const ApplyFAQ = () => {
         <h2 className="font-semibold md:text-[32px] xs:text-[22px]">What's next?</h2>
         <div className="flex flex-col gap-5">
           <h3 className="section-heading">
-            Learn more about DTI's core values and processes below.
+            Learn more about DTI's core values and application below.
```

Comment:
I thought it was just supposed to be "Learn more about DTI's application below." but if you think we talk about values inside the FAQ then I think it's fine

Reply:
oh you are right, i will update